### PR TITLE
Fix QA sample

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Try to address the issue about `PT_QUESTION_ANSWERING_SAMPLE` discussed in the following 2 comments:

https://github.com/huggingface/transformers/pull/16523#issuecomment-1088635272

https://github.com/huggingface/transformers/pull/16523#issuecomment-1090346578

### CODE REVIEW

**Comment by ydshieh** [comment]
@patrickvonplaten @sgugger  

I would like to have a feedback to see if this is in good direction.

@sgugger `make style` will break the line into 2 line. But when replacing the `qa_target_start_index` and `qa_target_end_index` by `14` & `15`, it actually fits in one line. Should I use a shorter name like `qa_target_start_idx` or `qa_start_index` for example?

**Comment by ydshieh** [comment]
cc @vumichien and @bhadreshpsavani to keep them updated

File: src/transformers/utils/doc.py [hunk 1]
```diff
@@ -207,7 +207,9 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
 
     ```python
     >>> # target is "nice puppet"
-    >>> target_start_index, target_end_index = torch.tensor([14]), torch.tensor([15])
+    >>> target_start_index, target_end_index = torch.tensor([{qa_target_start_index}]), torch.tensor(
+    ...     [{qa_target_end_index}]
+    ... )
```
Comment (sgugger) [inline src/transformers/utils/doc.py:212]: You should be on two separate lines since there is formatting.

**Comment by ydshieh** [comment]
I should do the same for `TF_QUESTION_ANSWERING_SAMPLE`. Currently just to have some feedbacks from you if any :-)

**Comment by ydshieh** [comment]
I also changed `TF_QUESTION_ANSWERING_SAMPLE`. 

Ran locally for `Roberta` and `TFRoberta`, and their doctest pass.

Ready for @sgugger to give a final look :-)

**Review by sgugger — APPROVED** [review]
Looks great, thanks!

**Pull request merged**